plat/rockchip/rk3368/drivers/pmu/pmu.c

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright (C) 2016, Fuzhou Rockchip Electronics Co., Ltd.
```

Comment:
I will keep the "ARM Limited and Contributors. All rights reserved." with the series patches